tests/testthat/test-load_files.R

```diff
@@ -15,7 +15,7 @@ context("load_files")
   # valid output returned for "manifest.xlsx"
   m_df <- load_manifest(td2$m_file)
   ref_m_df <- readRDS(td2$ref_m_df)
-  expect_identical(m_df, ref_m_df)
+  expect_equal(m_df, ref_m_df)
```

Comment:
Why do we get rid of `expect_identical`?

Reply:
Now we have:

`m_df` not identical to `ref_m_df`.
Objects equal but not identical

So I decided to keep only equal